recipes/numcpp/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools
+import os
+import glob
+from conans.errors import ConanInvalidConfiguration
+
+
```

Comment:
```suggestion
required_conan_version = ">=1.32.0"

```

validate() is Conan 1.32.0